packages/components/src/Combobox/components/Action/Action.css

```diff
@@ -0,0 +1,12 @@
+.action {
+  width: 100%;
+  padding: var(--space-small) var(--space-base);
+  border: none;
+  text-align: left;
+  background-color: var(--color-surface);
+}
+
+.action:hover,
+.action:focus {
+  background-color: var(--color-surface--hover);
+}
```

Comment:
consistency wise, no focus glow? 
![image](https://github.com/GetJobber/atlantis/assets/15986172/7c188ac4-71f2-4c3f-8d12-043ac8b5a057)

Reply:
pls ignore styles for now, we have a follow up subtask to make everything beautiful for real!

this current stuff is just from our PoC

Reply:
ayt! Consider it ignored!